dev/tasks/r/azure.linux.yml

```diff
@@ -47,6 +47,9 @@ jobs:
         export R_ORG={{ r_org }}
         export R_IMAGE={{ r_image }}
         export R_TAG={{ r_tag }}
+        export ARROW_DATASET={{ arrow_dataset|default("ON") }}
+        export ARROW_PARQUET={{ arrow_parquet|default("ON") }}
```

Comment:
Unfortunately this didn't pass through all the way: https://dev.azure.com/ursacomputing/crossbow/_build/results?buildId=1721&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=6c939d89-0d1a-51f2-8b30-091a7a82e98c&l=240

I believe what you instead want is to pass them into docker-compose below, something like `docker-compose -e ARROW_DATASET={{ arrow_dataset|default("ON") }} -e ARROW_PARQUET={{ arrow_parquet|default("ON") }} -e ARROW_S3={{ arrow_s3|default("ON") }} r`

Reply:
Got it, thanks. Let's see if it works to add them to `.env` (066ce065124cf7dcaa08225cae83177594da3965) and if not then I'll do it with `-e`.

Reply:
Oops, need to append not overwrite. Fixed in 318371fbaa07f1b6fcdf71f58d7d5460a931da13

Reply:
I think the only danger to appending to the .env file is that if you ran this locally, wouldn't it modify the file locally?

Reply:
Ok, if folks might be running these commands locally then I can see how that would cause damage.
Looks like it didn't work anyhow. I will investigate.

Reply:
Ah yeah, I think it's the same problem--note (for comparison) that `R_ORG` et al. are in the .env file, and they're environment variables for docker-compose, not passed into the docker container.

The fact that `ARROW_R_DEV` is in the env file, and then passed through in docker-compose.yml to the build environments, may just be due to my ignorance of all of this at the time I added it.

Reply:
Ok, thanks. I tried a few different things just for kicks, then used `-e` per your suggestion in 9479c6ce079c7230b0e290dcb5658436d1d62fde

Reply:
The build looks good after 9479c6c:
https://dev.azure.com/ursacomputing/crossbow/_build/latest?definitionId=1&branchName=actions-184-azure-test-r-minimal-build